syntax = "proto3";

message LocationResMessage {
  int32 person_id = 1;
  string longitude = 2;
  string latitude = 3;
  string start_date = 4;
  string end_date = 5;
}

message LocationReqMessage {
  string person_id = 1;
  string start_date = 2;
  string end_date = 3;
}

message LocationMessageList {
 repeated LocationResMessage locations = 1;
}

service LocationService {
   rpc Get(LocationReqMessage) returns (LocationMessageList);
}